components/brave_vpn/brave_vpn_service.h

```diff
@@ -164,7 +171,14 @@ class BraveVpnService :
                                   const std::string& payments_environment,
                                   const std::string& monthly_pass);
 
+  void RecordP3A(bool new_usage);
```

Comment:
It's not clear what `new_usage` means here. How about `RecordP3ANewConnection()` which does the new usage check and then calls a private `RecordP3A()` which handles the false case?

Reply:
we could, but what if I just left a comment above line 174?

Reply:
That would be better than nothing, but where I really want the comment is at foreign call sites. You could also comment just above brave_vpn_native_worker.cc:251 and whatever the equivalent desktop call site ends up being, but a more clear name without the mysterious bool seems easier.

Reply:
added comments, hopefully it's more clear